lib/matplotlib/backends/backend_pdf.py

```diff
@@ -1194,6 +1196,23 @@ def embedTTFType42(font, characters, descriptor):
             wObject = self.reserveObject('Type 0 widths')
             toUnicodeMapObject = self.reserveObject('ToUnicode map')
 
+            print(f"SUBSET {filename} characters: "
```

Comment:
turn these into logs?

Reply:
yeah, was waiting for a general review before changing the previous PR's contents